### PR TITLE
add a comment about non-HTTP protocols

### PR DESCRIPTION
**Pull request by grishick**
to address the concern raised on BZ : https://bugzilla.zimbra.com/show_bug.cgi?id=108217, add text to clarify that Jetty DosFilter applies only to HTTP/HTTPS

### CODE REVIEW

File: monitoring.adoc [hunk 1]
```diff
@@ -240,10 +240,10 @@ with `iostat -x`).
 == Configuring Denial of Service Filter Parameters
 
 The denial-of-service filter (DoSFilter) limits exposure to requests
-flooding.  The DoSFilter throttles clients sending a large number of
+flooding over HTTP/HTTPS.  The DoSFilter throttles clients sending a large number of
 requests over a short period of time.
 
-The DoSFilter is enabled by default on {product-abbrev} and is applied to all requests.
+The DoSFilter is enabled by default on {product-abbrev} and is applied to all HTTP and HTTPS requests. DosFilter dis not applied to POP3, IMAP or SMTP requests.
```
Comment (plobbes) [inline monitoring.adoc]: nit: typo ```dis``` → ```is```
- we could also reference zimbraPasswordLockout* attributes if we want to point them in something else they may be interested in

Comment (grishick) [reply monitoring.adoc]: I added a reference to the section that describes password lockout policy. Unfortunately, we do not have a section that describes how to set password lockout policy via CLI and related LDAP attributes and I think adding that section is out of scope of this bug.

Comment (fatkudu) [reply monitoring.adoc]: I'd suggest future proofing this.  e.g. Use "DosFilter is only applied to HTTP and HTTPS requests, in other words, it does **not** affect requests for any other protocols like POP3, IMAP or SMTP."

**Comment by grishick** [comment]
@fatkudu I've rephrased the paragraph about DoSFilter the way you suggested.

**Pull request merged**